canonical-web-and-design/www.ubuntu.com
# 5133 Starting with AI Takeover & Engage

## Done

Add starting with AI takeover and landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/) and [/engage/starting-with-ai](http://0.0.0.0:8001/engage/starting-with-ai)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure it complies with the [design](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5133#event-2333142188)
- Make sure it complies with the [brief](https://docs.google.com/document/d/1Wk1cowPs55rhNhbFCMPeSP-ynoxVP4hfH7ICg0T9X0E/edit#)


## Issue / Card

Fixes #5133
